src/widget.py

```diff
@@ -1,30 +1,23 @@
 from src.masks import get_mask_account, get_mask_card_number
-'''импортировал из маск функции'''
 
+"""импортировал из маск функции"""
 
-def mask_account_card(account_card: str) -> str :
 
+def mask_account_card(account_card: str) -> str:
     """скрываем счет и номер карты с помощью функций"""
 
-    if "счет" in account_card:
-        number_card = account_card[-10:]
-        masked_card = get_mask_account(number_card)
+    if "Счет" == account_card[:4]:
```

Comment:
Можно не брать срез, а сделать чуть проще:
```suggestion
    if account_card.startswith("Счет"):
```